Add dependency snippets to relevant doc sections

raboof opened the pull request:
As in #24232

raboof committed:
Add dependency snippets to relevant doc sections

raboof commented:
Hm again `sbtwhitesource.WhiteSourceException: Error communicating with service: Unexpected error. Response data is: Bad Gateway`

raboof commented:
PLS BUILD

raboof commented:
PLS BUILD

johanandren reviewed (approved):
LGTM, thanks!

patriknw reviewed (commented):
I think it's great to add this to all pages introducing a module/feature.

However, we should add it in a consistent way. Same heading, same location. 

I think it should be located as the first level 2 heading after the intro text that is typically at the top below the level 1 heading.

Heading name? "Dependency"?

raboof committed:
Add 'Dependency' headings

Tried to consistently add them to each section introducing a module, after
the introduction but before the first code sample.

patriknw commented on akka-docs/src/main/paradox/cluster-sharding.md:
```diff
@@ -31,6 +31,16 @@ See @ref:[Downing](cluster-usage.md#automatic-vs-manual-downing).
 
+## Dependency
+
+To use cluster sharding, add the module to your project:
```
nitpick: we often write module names like Cluster Sharing

patriknw commented on akka-docs/src/main/paradox/cluster-usage.md:
```diff
@@ -2,34 +2,15 @@
-## Preparing Your Project for Clustering
+## Dependency
 
 The Akka cluster is a separate jar file. Make sure that you have the following dependency in your project:
```
I think the wording should be exactly the same for all these

To use Akka Cluster, add the module to your project:

patriknw reviewed (commented):
consistency is king... ;-)

raboof committed:
Make Dependency sections more consistent

patriknw reviewed (approved):
LGTM, thanks
